tests/test_storage_manager.py

```
@@ -1502,5 +1502,141 @@ def test_save_with_metadata_includes_details_for_inconsistent_columns(self):
+        # Act
+        result = self.git_handler.is_git_repo()
+
+        # Assert: 例外をキャッチしてFalseを返す (lines 120-121)
+        self.assertFalse(result)
+
+    def test_is_all_zero_data_unicode_decode_error(self):
+        """_is_all_zero_data()でUnicodeDecodeErrorが発生した場合の処理を確認"""
+        # Arrange: デコードに失敗するデータをモック
+        with patch.object(self.storage, "_is_all_zero_data") as mock_check:
+            # 実際の実装を呼び出すが、内部でUnicodeDecodeErrorを発生させる
+            def raise_unicode_error(data):
```

Ruff (0.14.10)

Unused function argument: `data`

(ARG001)